app/services/fees/add_on_service.rb

```diff
@@ -21,6 +21,7 @@ def create
         invoiceable: applied_add_on,
         vat_rate: customer.applicable_vat_rate,
         units: 1,
+        payment_status: :pending,
```

Comment:
that's the default value, we can probably remove that line, no?

Reply:
I prefer to keep it to make it clear the default value is there and always `pending`